zyte_api/utils.py

```diff
@@ -42,10 +44,3 @@ def _process_query(query):
     if url == safe_url:
         return query
     return {**query, "url": safe_url}
-
-
-def user_agent(library):
-    return 'python-zyte-api/{} {}/{}'.format(
-        __version__,
-        library.__name__,
-        library.__version__)
```

Comment:
Hmm… I wonder if we should deprecate this instead of removing it, i.e. log a deprecation warning when the function is called, pointing to USER_AGENT as the replacement.

Reply:
We decided to drop it.